platforms/hermes-python/hermes_python/hermes.py

```diff
@@ -299,6 +300,17 @@ def disable_sound_feedback(self, site_message):
         self.ffi.sound_feedback.publish_toggle_off(site_message)
         return self
 
+    def register_sound(self, sound):
+        # type: (RegisterSoundMessage) -> Hermes
+        """
+        Register a sound that can later be played by the TTS.
+
+        :param sound: a sound to be played by the TTS.
+        :return: the current instance of Hermes to allow chaining.
+        """
+        self.ffi.tts.publish_register_sound(sound)
```

Comment:
Just curious, what happen if the file is not correctly registered? 
It could happen:
- if the audio file format is incorrect.
- if the length is above the requirements (<11 seconds))

Reply:
Every call made to a FFI function is checked and if there's an error, an exception is raised.

Reply:
I'll give it a try though

Reply:
I investigated with @fredszaq  and something I missed is that the `register_sound` FFI function is asynchronous.

The `snips-tts` component responds asynchronously. We need a callback that informs the client code if the sound was successfully registered.

While this could be added to the `hermes-protocol` it feels like a patch / an ad-hoc solution. 

For now, the proposed solution would be to use `snips-watch` to monitor the successful registration of a sound.

Reply:
:+1: